src/notes.ts

```diff
@@ -1,267 +0,0 @@
-// Setting application state and view updates are done via a Proxy
-import state from "./notes/state/index";
-
-import { newNote, openOptions, content, syncNow } from "./notes/view/elements";
-import typing from "./notes/typing";
-import toolbar from "./notes/toolbar/index";
-import { saveNotes, syncNotes } from "./notes/saving";
-import hotkeys from "./notes/hotkeys";
-import sidebar from "./notes/sidebar";
-import { newNoteModal } from "./notes/modals";
-import contextMenu from "./notes/context-menu";
-
-import notesHistory from "./notes/history";
-import { Message, MessageType, ContextMenuSelection, NotesObject } from "shared/storage/schema";
-import { sendMessage } from "messages/index";
-
-let tabId: string; // important so can update the content in other tabs (except the tab that has made the changes)
-chrome.tabs.getCurrent((tab) => {
-  if (!tab) {
-    return;
-  }
-
-  // set as a "string" to quickly compare with localStorage.getItem("notesChangedBy")
-  tabId = String(tab.id);
-
-  // Typing, Toolbar, Hotkeys, Sidebar
-  typing.initialize(content, tabId);
-  toolbar.initialize(content, tabId);
-  hotkeys.register(state);
-  sidebar.register();
-});
-
-// Create a New note
-newNote.addEventListener("click", () => {
-  newNoteModal((newNoteName) => {
-    state.createNote(newNoteName);
-  });
-});
-
-// Open Options
-openOptions.addEventListener("click", () => {
-  chrome.tabs.create({ url: "/options.html" });
-});
-
-// Sync Now
-syncNow.addEventListener("click", () => syncNotes(state));
```

Comment:
The whole state was passed before, although only its "sync" property was needed. This is now fixed.